Students/Benny/3. Protocols and bridging/Questions.md

```diff
@@ -20,3 +28,7 @@
 			[self.collectionView reloadData];
 		}];
 ```
+
+3. Explain the difference between NSOperation and GDC please.
```

Comment:
This is just a different set of API. GCD (Grand Central Dispatch) (not GDC). Is the lower level. NSOperation is a layer above this. It's main difference is:

1. Easier to add interdependencies between operations
2. You can stop an ongoing operation
3. Object oriented api instead of a procedural api
4. Used for inter operation communication. Operations can communicate from a different queue to another queue while still in process. This is more advanced usage.